bootstrap/httpserver/server.go

```diff
@@ -18,11 +18,11 @@ type HttpServer interface {
 
 type server struct {
 	httpServer *http.Server
-	reporting  instrumentation.Reporting
+	reporting  BasicLogger
 	publicApi  services.PublicApi
 }
 
-func NewHttpServer(address string, reporting instrumentation.Reporting, publicApi services.PublicApi) HttpServer {
+func NewHttpServer(address string, reporting BasicLogger, publicApi services.PublicApi) HttpServer {
```

Comment:
if we follow the same convention as in services, the logger for `server` should be `reporting.For("http-server")`. This would also help with the metering, as the name of the meter would be `request-process-time`, with the `http-server` prefix coming from the parent logger.